README.md

```diff
@@ -541,3 +542,28 @@ There is a WIP GUI which can be started via (from the root folder) `cargo r --bi
 
 It's quite barebones right now, but the goal is the be able to attach to any running Serial Keel instance and see the
 user events and also the raw logs from each endpoint in real time.
+
+## Serial Keel as a systemd service
+
+You can setup Serial Keel to run as a systemd service on startup. In `scripts/systemd`, you'll find a template `.service`
+file and an `install_serial_keel.sh` script.
+
+Running the install script with the template file, specified branch (defaults to `main` if not specified) and path to the config file
+will create a user service. It will also build serial-keel and install it.
+
+You can invoke this script as below:
+```
+./scripts/systemd/install_serial_keel.sh <branch-name> <path-to-config-file>
+```
+
+- Once done, the serial-keel service should be running. Running `systemctl --user status serial-keel.service`
+should let you know whether the service started successfully.
+
+- You can access the latest logs of the serial-keel service with:
+```
+journalctl --user-unit serial-keel.service -e
+```
```

Comment:
```suggestion
- Logs are accessed via `journalctl`. Check `man journalctl` for usage. Here is an example:
```
journalctl --user-unit serial-keel --pager-end --since "2 days ago" --output=cat
```
The `--output=cat` is a "raw" logging mode, this ensures we don't get timestamps twice.
```